src/backend/meson.build

```diff
@@ -183,7 +183,9 @@ pg_mod_args = default_mod_args + {
   'link_depends': pg_mod_link_depend,
 }
 
-
+pg_mod_args_no_install = pg_mod_args + {
+  'install': false
+}
 
```

Comment:
Should leave the amount of space to the next section "intact"

Reply:
Also, I'd rather name this pg_test_mod_args or such - the no_install bit describes the effect it has, rather than what it's for.